share/spice/crunchbase/collections/crunchbase_collections.js

```diff
@@ -0,0 +1,66 @@
+(function (env) {
+    "use strict";
+    env.ddg_spice_crunchbase_collections = function(api_result){
+        // throw error if no result, error result, no items, or array of 0 items
+        if (!api_result || api_result.error || !DDG.getProperty(api_result, 'data.items') || api_result.data.items.length === 0) {
+            return Spice.failed('crunchbase');
+        }
+
+        // return items with query in name
+        function filterVerbatim(item){
+            return item.properties.name.toLowerCase().indexOf(query) !== -1;
+        }
+        var data = api_result.data, // get org data
+	    items = data.items; // get all orgs
+        
+        var reCompany = /crunchbase (.*)/; // all words proceeding crunchbase
+        var query = DDG.get_query().match(reCompany)[1]; // get org query
+        var verbatimMatches = items.filter(filterVerbatim); // filter by query
+        
+        if (verbatimMatches){ // if matches by query 
+            var items = verbatimMatches; // limit items to those matches
+        }
+
+	    var metadata = api_result.metadata; // get metadata for base urls
```

Comment:
nit: there's a hard-tab here, @jagtalon will declare war if it's not switched out to 4 spaces!